docs/auth/azure-key-vault.md

```diff
@@ -3,11 +3,11 @@ title: "Authentication for Azure Key Vault"
 layout: default
 ---
 
-## Authentication
+# Authentication
```

Comment:
Yes, not sure what the 'guidelines' say about this. I think there are other pages to (maybe on ohter repo's) that also uses the second header (`## `) as initlal header.

Reply:
That is correct but I've aligned Web API already - https://github.com/arcus-azure/arcus.webapi/pull/57/files#diff-9b0c7b7749e7052a0df13b318231d12cR6

Reply:
I think this makes more sense unless you disagree?

Can revert it, no problem.

Reply:
Will merge and send new PR if it has to be changed.

Reply:
No, it's good like it is. Just a question I had for previous and future docs.